CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ This name should be decided amongst the team before the release.
 ### Fixed
 - [#779](https://github.com/tweag/topiary/pull/779) Load relevant grammars before CLI tests
 - [#799](https://github.com/tweag/topiary/pull/799) Line break after table-less pairs in TOML
+- tree-sitter-query: Long predicate parameters now retain newline separation
+- tree-sitter-query: Prevent invalid concatenation of `field_definition`s that followed an `identifier`
```

Comment:
Usually we bundle fixes to language formatting into one line on the changelog. As you've done the work, you'd get credit too :+1:

Something like this:

```suggestion
- [#826](https://github.com/tweag/topiary/pull/799) Various Tree-sitter query fixes, thanks to @mkatychev
```